components/clm/src/dyn_subgrid/dynHarvestMod.F90

```diff
@@ -94,64 +119,64 @@ subroutine dynHarvest_init(bounds, harvest_filename)
 
     SHR_ASSERT_ALL(bounds%level == BOUNDS_LEVEL_PROC, subname // ': argument must be PROC-level bounds')
```

Comment:
was going to check if this code is thread-safe seeing how a global (harvest_rates) is being allocated, but both the initialization and the interp code asserts this is the proc level. 
check